UnitedIncome/components
# [APP-2970] Adds saveToContext to MCG

### Description
> Please provide a brief description of your changes below.

Adds the `saveToContext` utility to McGonnagallalallal.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* This can be used in a flow like so:

```javascript
import {saveToContext} from '@unitedincome/components/dist/mcgonagall';
```

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-2970)
